branchpro/data_library/covid_ontario/parse_data.py

```diff
@@ -0,0 +1,100 @@
+#
+# This file is part of BRANCHPRO
+# (https://github.com/SABS-R3-Epidemiology/branchpro.git) which is released
+# under the BSD 3-clause license. See accompanying LICENSE.md for copyright
+# notice and full license details.
+#
+
+"""Processing script for Ontario State data from [1]_.
+
+It splits the cases by state, selects certain dates, and rewrites the data file
+into the format expected by our app.
+
+References
+----------
+.. [1] Price, David J., et al. "Early analysis of the Australian COVID-19
+       epidemic." Elife 9 (2020): e58785.
+       https://elifesciences.org/articles/58785/figures#content
```

Comment:
We need to update the reference. I would include a link to the website we used and also mention what day we downloaded the file.